build.sh

```diff
@@ -19,7 +19,8 @@ join -t , -1 2 -2 2 -o 1.1,0,2.3,2.4 <(sort -k 2 -t , target/pub_postcodes.csv)
 awk -F, -f filter_postcodes.awk target/named_pubs.csv | awk -F, -f split_pubs.awk
 
 #Add additional bits to "outcodes" for non postcode places
-awk 'BEGIN { FS="\t"; OFS=","; } $8 ~ /^(PPL|ADM)/ && $5 > 49 && $5 < 61 && $6 > -8 && $6 < 2 { gsub(/[^a-zA-Z0-9]/, "", $2); print $8, $2, $5, $6 }' GB.txt | tr '[:lower:]' '[:upper:]' | sort -u -k2,2 -t, | awk -F, -f split_locations.awk
+awk 'BEGIN { FS="\t"; OFS=","; } $8 ~ /^(PPL|ADM)/ && $5 > 49 && $5 < 61 && $6 > -8 && $6 < 2 { gsub(/[^a-zA-Z0-9]/, "", $2); print $8, $2, $5, $6, $15 }' GB.txt \
+| sort -rnk5,5 -t, | awk -F, 'BEGIN { OFS = FS }; NF { NF -= 1 }; 1' | tr '[:lower:]' '[:upper:]' | sort -u -k2,2 -t, | awk -F, -f split_locations.awk
```

Comment:
OBSERVATION: Would the following be a bit neater?

```suggestion
| sort -u -k2,2 -rnk5,5 -t, | cut -f1,2,3,4 -d, | awk -F, -f split_locations.awk
```